new-frontiers-14/frontier-station-14
# Smuggling Revamp: The Syndicate Broker

## About the PR


Adds 2 new jobs: The Syndicate Broker and Syndicate Bodyguard a soft antag group in charge of distributing dead drops from a POI they commandeer.
I wrote a more exhaustive description on the discord ideas section under the same name to prevent cluttering this PR.

Reworked the deaddrop system to allow printing jobs from fax machines that have the component while still allowing dead drop posters to exist.

Also as a note, the broker starting with cash is a band-aid fix until those NFSD ship vouchers get added because then I can just reskin one for the broker. The cheapest ship there costs 50K and sells for far less.

## Why / Balance

Smuggling as it is, is an RP wasteland, for both smugglers and NFSD. Not only that, making smuggling have greater ramifications for the sector as well as soft antag syndie jobs have been discussed and wished for ad-nauseam.
## How to test

Boot server and join as the Syndicate Broker, observe the new fax machine they start with. 
## Media

![broker](https://github.com/new-frontiers-14/frontier-station-14/assets/92019614/d2d576d9-5823-4105-8651-ac03c0989f3f)
![broker2](https://github.com/new-frontiers-14/frontier-station-14/assets/92019614/293f9f6f-173b-4bd2-a2af-db96cd18c2d1)


## Breaking changes


**Changelog**




add: The Syndicate is now hiring for a new "public relations" position at listening point bravo.
